src/omd.ml

```diff
@@ -14,6 +14,41 @@ let parse_inlines (md, defs) =
   in
   List.map (Mapper.map (parse_inline defs)) md
 
+let txt ?(attrs = []) s = Text (attrs, s)
+
+let em ?(attrs = []) il = Emph (attrs, il)
+
+let strong ?(attrs = []) il = Strong (attrs, il)
+
+let code ?(attrs = []) s = Code (attrs, s)
+
+let hard_break attrs = Hard_break attrs
+
+let soft_break attrs = Soft_break attrs
```

Comment:
Would that be better to have one `br` constructor with a `soft` argument? Something like:

```ocaml
val br : ?soft:bool -> attributes -> attributes inline
```